pkg/syncer/syncer.go

```diff
@@ -291,6 +292,12 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 		return err
 	}
 
+	logger.Info("Creating resource upsyncer")
```

Comment:
does this need log output?

Reply:
We have messages for the other 2 controllers. Why not be consistent ?
These are only output at the start of the Syncer anyway.